backend/Auth/Authentication/views.py

```
@@ -0,0 +1,127 @@
+    """Handle user registration with token creation"""
+    def post(self, request):
+        serializer = UserRegistrationSerializer(data=request.data)
+        
+        if serializer.is_valid():
+            try:
+                serializer.save()
+                return Response(
+                    {'message': 'User created successfully'},
+                    status=status.HTTP_201_CREATED
+                )
+            except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)